emerald-lang-spec/5if-else.em

```diff
@@ -0,0 +1,17 @@
+(if (< 2 20)
+  print "val1 is less than val2"
```

Comment:
if takes 3 arguments, you have 4 here as you're not calling print as a function, it's just the atom "print".